lib/utils.js

```diff
@@ -147,7 +147,14 @@ var canonicalType = (exports.canonicalType = function canonicalType(value) {
     return 'null';
   } else if (Buffer.isBuffer(value)) {
     return 'buffer';
+  } else if (
+    typeof value === 'object' &&
+    // eslint-disable-next-line no-prototype-builtins
+    !Object.prototype.isPrototypeOf(value)
+  ) {
```

Comment:
We can probably do this instead? And avoid having to opt out of the eslint rules?

```suggestion
  } else if (Object.getPrototypeOf(value) === null) {
```

Reply:
Yup that is cleaner, pushed a small update